### ISSUE TITLE
go mod tidy warns about missing dependencies

### ISSUE THREAD

**Issue by dubek** [IBM/ibm-cos-sdk-go#3]
### Version of AWS SDK for Go?

v1.2.0

### Version of Go (`go version`)?

```
go version go1.13.1 linux/amd64
```

### What issue did you see?

`go mod tidy` prints these warnings:

```
github.com/IBM/ibm-cos-sdk-go/service/cloudfront: module github.com/IBM/ibm-cos-sdk-go@latest (v1.2.0) found, but does not contain package github.com/IBM/ibm-cos-sdk-go/service/cloudfront
```

### Steps to reproduce

```
dubek@vm-ubuntu ~/testcos $ go mod init example.com/testcos
go: creating new go.mod: module example.com/testcos


dubek@vm-ubuntu ~/testcos $ cat go.mod
module example.com/testcos

go 1.13


dubek@vm-ubuntu ~/testcos $ cat main.go
package main

import (
        "fmt"
        _ "github.com/IBM/ibm-cos-sdk-go/service/s3"
)

func main() {
        fmt.Print("Hello")
}


dubek@vm-ubuntu ~/testcos $ go mod tidy
go: finding golang.org/x/net latest
go: downloading golang.org/x/net v0.0.0-20191009170851-d66e71096ffb
go: extracting golang.org/x/net v0.0.0-20191009170851-d66e71096ffb
go: downloading github.com/davecgh/go-spew v1.1.0
go: downloading golang.org/x/text v0.3.0
go: downloading gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405
go: extracting github.com/davecgh/go-spew v1.1.0
go: extracting gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405
go: extracting golang.org/x/text v0.3.0
example.com/testcos imports
        github.com/IBM/ibm-cos-sdk-go/service/s3 imports
        github.com/IBM/ibm-cos-sdk-go/private/protocol/restxml tested by
        github.com/IBM/ibm-cos-sdk-go/private/protocol/restxml.test imports
        github.com/IBM/ibm-cos-sdk-go/service/cloudfront: module github.com/IBM/ibm-cos-sdk-go@latest (v1.2.0) found, but does not contain package github.com/IBM/ibm-cos-sdk-go/service/cloudfront
example.com/testcos imports
        github.com/IBM/ibm-cos-sdk-go/service/s3 imports
        github.com/IBM/ibm-cos-sdk-go/aws/request tested by
        github.com/IBM/ibm-cos-sdk-go/aws/request.test imports
        github.com/IBM/ibm-cos-sdk-go/private/protocol/jsonrpc tested by
        github.com/IBM/ibm-cos-sdk-go/private/protocol/jsonrpc.test imports
        github.com/IBM/ibm-cos-sdk-go/service/dynamodb: module github.com/IBM/ibm-cos-sdk-go@latest (v1.2.0) found, but does not contain package github.com/IBM/ibm-cos-sdk-go/service/dynamodb
example.com/testcos imports
        github.com/IBM/ibm-cos-sdk-go/service/s3 imports
        github.com/IBM/ibm-cos-sdk-go/aws/request tested by
        github.com/IBM/ibm-cos-sdk-go/aws/request.test imports
        github.com/IBM/ibm-cos-sdk-go/private/protocol/jsonrpc tested by
        github.com/IBM/ibm-cos-sdk-go/private/protocol/jsonrpc.test imports
        github.com/IBM/ibm-cos-sdk-go/service/dynamodb/dynamodbattribute: module github.com/IBM/ibm-cos-sdk-go@latest (v1.2.0) found, but does not contain package github.com/IBM/ibm-cos-sdk-go/service/dynamodb/dynamodbattribute
example.com/testcos imports
        github.com/IBM/ibm-cos-sdk-go/service/s3 imports
        github.com/IBM/ibm-cos-sdk-go/private/protocol tested by
        github.com/IBM/ibm-cos-sdk-go/private/protocol.test imports
        github.com/IBM/ibm-cos-sdk-go/private/protocol/ec2query tested by
        github.com/IBM/ibm-cos-sdk-go/private/protocol/ec2query.test imports
        github.com/IBM/ibm-cos-sdk-go/service/ec2: module github.com/IBM/ibm-cos-sdk-go@latest (v1.2.0) found, but does not contain package github.com/IBM/ibm-cos-sdk-go/service/ec2
example.com/testcos imports
        github.com/IBM/ibm-cos-sdk-go/service/s3 imports
        github.com/IBM/ibm-cos-sdk-go/private/protocol tested by
        github.com/IBM/ibm-cos-sdk-go/private/protocol.test imports
        github.com/IBM/ibm-cos-sdk-go/private/protocol/restjson tested by
        github.com/IBM/ibm-cos-sdk-go/private/protocol/restjson.test imports
        github.com/IBM/ibm-cos-sdk-go/service/elastictranscoder: module github.com/IBM/ibm-cos-sdk-go@latest (v1.2.0) found, but does not contain package github.com/IBM/ibm-cos-sdk-go/service/elastictranscoder
```

**Comment by wbhuberIBM** [comment]
@dubek - while you receive warning messages from running ```go mod tidy``` using Go version 1.13, does that not inhibit you from running S3 APIs using this SDK?  We released this SDK version 1.2.0 around the same time Go 1.13 was released.  We will support 1.13 in the next release and fix those warning messages by then.

Internal ticket to track this: CSAFE-62178

**Comment by wbhuberIBM** [comment]
@dubek - can you please verify the fix on Go SDK 1.3.0 that's been released last week?  Thanks!

**Comment by dubek** [comment]
@wbhuberIBM - thanks, now it works perfectly:

```
$ go mod tidy
go: finding github.com/IBM/ibm-cos-sdk-go v1.3.0
go: downloading github.com/IBM/ibm-cos-sdk-go v1.3.0
go: extracting github.com/IBM/ibm-cos-sdk-go v1.3.0
```

I can confirm it is resolved in v1.3.0.
